recipes/zlib/1.2.11/conanfile.py

```diff
@@ -69,7 +68,7 @@ def _build_zlib_autotools(self):
         if "clang" in str(self.settings.compiler) and tools.get_env("CC") is None and tools.get_env("CXX") is None:
             env = {"CC": "clang", "CXX": "clang++"}
         with tools.environment_append(env):
-            env_build.configure("../", build=False, host=False, target=False)
+            env_build.configure("../")
```

Comment:
please don't change, it's here for the reason - configure of zlib doesn't support `--build`, `--host` or `--target` arguments, build will fail if they are used.

Reply:
ok, will revert these line